api/src/main/java/marquez/db/mappers/DatasetMapper.java

```diff
@@ -88,12 +93,13 @@ public Dataset map(@NonNull ResultSet results, @NonNull StatementContext context
           toTags(results, "tags"),
           timestampOrNull(results, Columns.LAST_MODIFIED_AT),
           stringOrNull(results, Columns.DESCRIPTION),
-          Optional.ofNullable(uuidOrNull(results, Columns.CURRENT_VERSION_UUID)));
+          Optional.ofNullable(uuidOrNull(results, Columns.CURRENT_VERSION_UUID)),
+          toFacetsOrNull(results));
     }
   }
 
   private URL getUrl(ResultSet results, String column) throws SQLException, MalformedURLException {
-    if (!hasColumn(results, column)) {
+    if (!Columns.columnExists(results, column)) {
```

Comment:
If the tendency is to always use the class name with the method, as opposed to a static import, I'd prefer to write it as `Columns.exists(results, column)`

Reply:
Sure, reduces the wordiness 👍